### PR TITLE
show the hand cursor on hover

### PR DESCRIPTION
**Pull request by jgilbertcastro**

### CODE REVIEW

File: src/assets/style/custom.css [hunk 1]
```diff
@@ -16,4 +16,7 @@
 body {
     background-color: #f5f6fa;
     color: #343a40;
+}
+.pointer-hand{
```
Comment (vyzaldysanchez) [inline src/assets/style/custom.css]: This could've been `scoped` to the component too, right @DannyFeliz ?

Comment (jgilbertcastro) [reply src/assets/style/custom.css]: yes, but this css class could be used by more than one component if necessary

Comment (vyzaldysanchez) [reply src/assets/style/custom.css]: @jgilbertcastro I agree, I just don't see the immediate use of it and I'm kind of a fan of adding things as reusable when knowing a concrete use case for it aside from the current one. (Refactoring when seeing duplication coming along)

**But**, if you and @DannyFeliz feel good about it, no problem here.

Comment (DannyFeliz) [inline src/assets/style/custom.css]:
```suggestion
.pointer-hand {
```

Comment (DannyFeliz) [reply src/assets/style/custom.css]: We can leave it like this for now, but please add a space between the class name and the open curly brace. @jgilbertcastro

**Comment by jgilbertcastro** [comment]
Done

**Comment by DannyFeliz** [comment]
> Done

I don't see any changes, what's different from the last request and now?

File: src/assets/style/custom.css (hunk 1 above)
Comment (vyzaldysanchez) [inline src/assets/style/custom.css]: I still see the missing space here between the class name and the bracket.

**Pull request closed without merging**